storage/vdi_automaton.ml

```diff
@@ -87,10 +87,10 @@ let ( + ) state operation =
 		| Activated _, Detach     -> error ()
 
 let superstate states =
-	let activated = List.fold_left (fun acc s ->
-		acc || (s = Activated RO) || (s = Activated RW)) false states in
-	let rw = List.fold_left (fun acc s ->
-		acc || (s = Activated RW) || (s = Attached RW)) false states in
+	let activated = List.exists (fun s ->
```

Comment:
The predicate could be further simplified: 
```
let activated = List.exists (function Activated _ -> true | _ -> false) states
```

Activated is true, if at least one activated state exists.